### PR TITLE
Add rule for checking non corresponding storage keeps unchanged

### PR DESCRIPTION
**Pull request by sunbreak1211**

### CODE REVIEW

**Review by oldchili — APPROVED** [review]
Reviewed the changes, making sure the affected places are correct but did not strictly examine every line.

Certora runs for all the specs pass.

LGTM.

File: certora/funnels/DepositorUniV3.spec [hunk 1]
```diff
@@ -609,9 +574,6 @@ rule collect(DepositorUniV3.CollectParams p) {
     address buffer = buffer();
     require buffer != poolCon;
 
-    mathint wardsBefore = wards(anyAddr);
```
Comment (telome) [inline certora/funnels/DepositorUniV3.spec:612]: The  `anyAddr`, `anyAddr_2` and `anyUint24` vars above are no longer used.

File: certora/funnels/Swapper.spec [hunk 2]
```diff
@@ -176,36 +177,32 @@ rule swap(address src, address dst, uint256 amt, uint256 minOut, address callee,
     require buffer != currentContract;
     require buffer != callee;
 
+    mathint a; mathint b;
+
```
Comment (telome) [inline certora/funnels/Swapper.spec:180]: `wardsBefore` below is no longer used

Comment (telome) [reply certora/funnels/Swapper.spec:180]: Same for `anyAddr` above

File: certora/funnels/DepositorUniV3.spec (hunk 1 above)
Comment (sunbreak1211) [reply certora/funnels/DepositorUniV3.spec:612]: Done!

File: certora/funnels/Swapper.spec (hunk 2 above)
Comment (sunbreak1211) [reply certora/funnels/Swapper.spec:180]: Done.

**Review by telome — APPROVED** [review]
LGTM!

**Pull request merged**